manifests/config.pp

```diff
@@ -1,5 +1,8 @@
 # Configure dns
 class dns::config {
+  validate_array($dns::forwarders)
```

Comment:
In general we place these in init.pp. I've noticed this module is lacking them at all. Would you mind moving them?